requirements.txt

```diff
@@ -0,0 +1,4 @@
+dwave-ocean-sdk>=3.3.0
+jupyter
+matplotlib~=3.0
+bokeh
```

Comment:
I'm guessing the bokeh version used before this repo was archived was 2.x because [helpers/draw.py](https://github.com/dwave-examples/diverse-solutions/blob/577bc30f556223101691535f003aaa9e538b04c0/helpers/draw.py#L25) uses ``from bokeh.models.graphs import from_networkx`` but that was [removed in bokeh 3.0](https://github.com/bokeh/bokeh/wiki/Migration-Guides#removals-): ``from_networkx function is now only available to import from bokeh.plotting.``